src/components/ui/shadcn-io/dropzone/index.tsx

```
@@ -0,0 +1,202 @@
+'use client';
```

ESLint

[error] Delete `;`

(prettier/prettier)